README.md

```diff
@@ -37,6 +40,28 @@ docker run -v ${PWD}:/tmp mawrkus/joey-the-differ -s /tmp/demo/source.json -t /t
 
 Have a look at the [demo folder](./demo) to see the content of the files.
 
+#### Bulk diffing
```

Comment:
Missing documentation about the output here

Reply:
Added